### PR TITLE
tags/shared/*.tag: change to 2-space indentation #7352

### LINKED ISSUE #7352: *.jsp: change to 2-space indentation
Refactor all *.jsp pages to use 2-space indentation instead of 4.

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352
Applied to all files in `src/main/webapp/WEB-INF/tags/shared/*`

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentAdd.tag [hunk 1]
```diff
@@ -10,25 +10,25 @@
...
-                                        secondIndex="${secondIndex}"
-                                        thirdIndex="${thirdIndex}"
-                                        fourthIndex="${fourthIndex}"
-                                        frc="${frc}"
-                                        divId="${divId}"
-                                        formType="Add"
-                                        textAreaId="responseCommentAddForm"
-                                        submitLink="${submitLink}"
-                                        buttonText="Add" />
+  id="showResponseCommentAddForm-${divId}" style="display: none;">
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentAdd.tag]: This line should be indented 4 spaces instead.

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag [hunk 2]
```diff
@@ -12,90 +12,91 @@
...
+        <a href="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACK_RESPONSE_COMMENT_DELETE %>"
+            type="button"
+            id="commentdelete-${divId}"
+            class="btn btn-default btn-xs icon-button"
+            data-toggle="tooltip"
+            data-placement="top"
+            title="<%= Const.Tooltips.COMMENT_DELETE %>"
+          <c:if test="${not frc.editDeleteEnabled}">
+            disabled
+          </c:if>>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag]: This if block should be indented an additional 2 spaces.

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag [hunk 3]
```diff
@@ -12,90 +12,91 @@
...
+        <input type="hidden" name="<%= Const.ParamsNames.SESSION_TOKEN %>" value="${data.sessionToken}">
+      </form>
+      <a type="button" id="commentedit-${divId}"
+          <c:choose>
+            <c:when test="${not empty firstIndex && not empty secondIndex && not empty thirdIndex && not empty frcIndex}">
+              class="btn btn-default btn-xs icon-button pull-right show-frc-edit-form"
+              data-recipientindex="${firstIndex}" data-giverindex="${secondIndex}"
+              data-qnindex="${thirdIndex}" data-frcindex="${frcIndex}"
+              <c:if test="${not empty fourthIndex}">data-sectionindex="${fourthIndex}"</c:if>
+             <c:if test="${not empty viewType}">data-viewtype="${viewType}"</c:if>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag]: Add an extra space here.

**Comment by sujeet14108** [comment]
@chowyb Ready for review

File: src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag [hunk 4]
```diff
@@ -7,52 +7,52 @@
...
-                    Submit Feedback
-                </button>
-            </c:otherwise>
-        </c:choose>
-    </div>
-    <br>
-    <br>
+    <c:choose>
+      <c:when test="${empty data.bundle.questionResponseBundle}">
+          There are no questions for you to answer here!
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag]: Is this indented too much?

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentForm.tag [hunk 5]
```diff
@@ -18,206 +18,206 @@
...
+            </td>
+            <td>
+              <input class="visibilityCheckbox giverCheckbox"
+                  type="checkbox"
+                  value="<%= FeedbackParticipantType.RECEIVER_TEAM_MEMBERS %>"
+                  <c:if test="${frc.showGiverNameToResponseRecipientTeam}">checked</c:if>>
+            </td>
+          </tr>
+        </c:if>
+         <c:if test="${frc.responseVisibleToStudents}">
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentForm.tag]: This should be one space to the left.

File: src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag (hunk 4 above)
Comment (sujeet14108) [inline src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag]: Done :sweat_smile:

File: src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentForm.tag (hunk 5 above)
Comment (sujeet14108) [inline src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentForm.tag]: Done

**Pull request merged**